tar/bsdtar.1

```diff
@@ -386,8 +386,7 @@ and the default behavior in c, r, and u modes or if
 .Nm
 is run in x mode as root.
 .It Fl n , Fl Fl norecurse , Fl Fl no-recursion
-(c, r, u modes only)
-Do not recursively archive the contents of directories.
+Do not recursively archive (c, r, u), extract (x) or list (t) the contents of directories.
```

Comment:
Maybe just: `Do not operate recursively on the content of directories.`

Reply:
Okay, done.